_includes/pbc/all/install-features/202505.0/ssp/install-ssp-asset-management-features.md

```diff
@@ -0,0 +1,457 @@
+            new UpdateSspAssetPermissionPlugin(),
+            new UnassignSspAssetPermissionPlugin(),
+            new CreateSspAssetPermissionPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Client/Permission/PermissionDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Client\Permission;
+
+use Spryker\Client\Permission\PermissionDependencyProvider as SprykerPermissionDependencyProvider;
+use SprykerFeature\Shared\SspAssetManagement\Plugin\Permission\CreateSspAssetPermissionPlugin;
+use SprykerFeature\Shared\SspAssetManagement\Plugin\Permission\UnassignSspAssetPermissionPlugin;
+use SprykerFeature\Shared\SspAssetManagement\Plugin\Permission\UpdateSspAssetPermissionPlugin;
+use SprykerFeature\Shared\SspAssetManagement\Plugin\Permission\ViewBusinessUnitSspAssetPermissionPlugin;
+use SprykerFeature\Shared\SspAssetManagement\Plugin\Permission\ViewCompanySspAssetPermissionPlugin;
+
+class PermissionDependencyProvider extends SprykerPermissionDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Shared\PermissionExtension\Dependency\Plugin\PermissionPluginInterface>
+     */
+    protected function getPermissionPlugins(): array
+    {
+        return [
+            new ViewCompanySspAssetPermissionPlugin(),
+            new ViewBusinessUnitSspAssetPermissionPlugin(),
+            new UpdateSspAssetPermissionPlugin(),
+            new UnassignSspAssetPermissionPlugin(),
+            new CreateSspAssetPermissionPlugin(),
+        ];
+    }
+}
+
+```
+
+**src/Pyz/Yves/Router/RouterDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\Router;
+
+use Spryker\Yves\Router\RouterDependencyProvider as SprykerRouterDependencyProvider;
+use SprykerFeature\Yves\SspAssetManagement\Plugin\Router\SspAssetRouteProviderPlugin;
+
+class RouterDependencyProvider extends SprykerRouterDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Yves\RouterExtension\Dependency\Plugin\RouteProviderPluginInterface>
+     */
+    protected function getRouteProvider(): array
+    {
+        return [
+            new SspAssetRouteProviderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+{% endinfo_block %}
+
+### Set up widgets
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerFeature\Yves\SspAssetManagement\Widget\SspAssetListWidget;
+use SprykerFeature\Yves\SspAssetManagement\Widget\SspAssetMenuItemWidget;
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @return array<string>
+     */
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            SspAssetMenuItemWidget::class,
+            SspAssetListWidget::class,
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
```

Comment:
same

Reply:
Verification was empty initially, I added it later.